Documentation/Administration/Docker/DockerComposeDemo/_codesnippets/_docker-compose.yaml

```
@@ -0,0 +1,28 @@
+version: '3.9'
+
+services:
+  db:
+    image: mariadb:10.6
+    container_name: compose-demo-typo3db
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: db
+      MYSQL_USER: db
+      MYSQL_PASSWORD: db
+    volumes:
+      - db_data:/var/lib/mysql
+
+  web:
+    image: martinhelmich/typo3:latest
+    container_name: compose-demo-typo3
```

```suggestion
    container_name: compose-demo-typo3
    # Using linux? Uncomment the line below and use CURRENT_UID=$(id -u):$(id -g) docker-compose up to run
    # user: ${CURRENT_UID}
```

I'm not sure about the location, but this way the user permissions are resolved. The `user` option switches the user to the current user id, e.g. `1000:1000`. This will give you a homeless, nameless user in the container. Which doesn't matter, but will help to prevent file issues. This fix works on all os, except for windows native. 

Same can be done in the docker command using `CURRENT_UID=$(id -u):$(id -g)  docker run --user=$CURRENT_UID`